src/view/RepeatedFragment.js

```diff
@@ -210,6 +210,10 @@ export default class RepeatedFragment {
 			if ( newIndex === -1 ) return;
 
 			const fragment = this.iterations[ oldIndex ];
+
+			// guard against null fragment - #2005 (feels like treating the symptom, not the cause?)
+			if ( fragment == null ) return;
```

Comment:
I normally use == to compare against null as my intent is to include undefined as well. Not sure how folks compare for null+undefined?